x11-misc/lightdm-gtk-greeter/lightdm-gtk-greeter-2.0.3-r1.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2017 Gentoo Foundation
```

Comment:
It's 2018 now.

Use `repoman commit` instead of git commit, and it'll take care of little things like this.